File: app/src/main/java/jgappsandgames/smartreminderslite/date/WeekAdapter.java
package jgappsandgames.smartreminderslite.date;

// App
import jgappsandgames.smartreminderslite.adapter.TaskAdapterInterface;

// Save
import jgappsandgames.smartreminderssave.date.DateManager;

/**
 * WeekAdapter
 * Created by joshua on 10/9/17.
 */
class WeekAdapter extends TaskAdapterInterface {
    // Initializer
    WeekAdapter(WeekActivity activity, int date_active) {
        super(activity, activity, DateManager.getWeek(date_active).getAllTasks());
    }
}

File: app/src/main/java/jgappsandgames/smartreminderslite/sort/date/WeekAdapter.kt
package jgappsandgames.smartreminderslite.sort.date

// App
import jgappsandgames.smartreminderslite.adapter.TaskAdapterInterface

// Save
import jgappsandgames.smartreminderssave.date.DateManager

/**
 * WeekAdapter
 * Created by joshua on 1/19/2018.
 */
class WeekAdapter(activity: WeekActivity, date_active: Int):
        TaskAdapterInterface(activity, activity, DateManager.getWeek(date_active).getAllTasks())